regex-assembly/951240.ra

```diff
@@ -8,11 +8,11 @@ pg_query\(\) \[:
 pg_exec\(\) \[:
 PostgreSQL.{1,20}ERROR
 ##! Example match: Warning: pg_fetch_array(): 3 is not a valid PostgreSQL result resource
-Warning.{1,20}\bpg_.*
+Warning.{1,20}\bpg_[^\r\n]*
```

Comment:
```suggestion
Warning.{1,20}\bpg_
```
Do we need this last part? It probably doesn't add too much to the matching itself...